beets/plugins.py

```diff
@@ -635,11 +635,11 @@ def get_artist(artists, id_key='id', name_key='name'):
 
         :param artists: Iterable of artist dicts returned by API.
         :type artists: list[dict]
-        :param id_key: Key corresponding to ``artist_id`` value.
-        :type id_key: str
-        :param name_key: Keys corresponding to values to concatenate
+        :param id_key: Key or index corresponding to ``artist_id`` value.
+        :type id_key: str or int
+        :param name_key: Key or index corresponding to values to concatenate
             for ``artist``.
```

Comment:
Seems like this description could use a little more detail. What exactly is `artist`, for example? (I think it's an element of the `artists` list.) Furthermore, it sounds like the `artists` list can actually contain dicts _or_ lists—is that right?

Reply:
Good catch, the artist list can indeed contain dicts or lists. Docstring expanded and updated accordingly.